build/yamls/base/conf/antrea-agent.conf

```diff
@@ -5,6 +5,11 @@ featureGates:
 # Service traffic.
 #  AntreaProxy: true
 
+# Enable EndpointSlice support in AntreaProxy. Don't enable this feature unless that EndpointSlice
```

Comment:
Probably better to mention the EndpointSlice API version here too.

Reply:
Agreed, it should be mentioned here.